SingularityS3Uploader/src/main/java/com/hubspot/singularity/s3uploader/config/SingularityS3UploaderConfiguration.java

```diff
@@ -42,15 +43,19 @@
   @Obfuscate
   private Optional<String> s3SecretKey = Optional.absent();
 
+  @Max(5368709120L)
```

Comment:
added a Min of 5242880L here, any smaller than that and there isn't a way multipart will work since that's also the minimum part size for multipart